libraries/botbuilder-ai/src/luis-client/luisClient.ts

```diff
@@ -51,12 +51,23 @@ export interface PredictionResolveOptionalParams {
     customHeaders: {headers: {[name: string]: string}};
 }
 
+ /**
+  *  Sets headers for request to LUIS service.
+  *  LUIS service uses the `Ocp-Apim-Subscription-Key` header. 
+  */
 export enum LuisApikeys {
     apiKeyHeader,
 }
 
+/**
+ * Luis Client Class
+ *
+ * @remarks
+ * This is a helper class for all the http request operations against the LUIS API.
+ * https://www.luis.ai/
+ */
 export class LuisClient {
-    private _basePath: string = '';
+    protected _basePath: string = '';
```

Comment:
@Konigstiger I think that when you rebased against _master_ branch you reverted the change that made this member `private`. Also, I would suggest to leave the comment about what this member is, and what are its allowed values (something like [this](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/luis/runtime/lib/lUISRuntimeClient.d.ts#L23) maybe?)

Reply:
I like that description, I will take your example. 
Also I will change the field to private too, as you said that was resolved before in other PR. Thanks!

Reply:
I like that description, I will take your example.
Also I will change the field to private too, as you said that was resolved before in other PR. Thanks!